test/js/acme.js

```diff
@@ -0,0 +1,89 @@
+var colors = require("colors");
+var crypto = require("./crypto-util");
+var request = require('request');
+function Acme(privateKey) {
+  this.privateKey = privateKey;
+  this.nonces = [];
+}
+
+Acme.prototype.getNonce = function(url, callback) {
+  var req = request.head({
+    url: url,
+  }, function(error, response, body) {
+    if (error) {
+      console.log(error);
```

Comment:
If you want this to show up highlighted red in browsers you can use `console.error(error)` here instead